python/lsst/ts/xml/data/sal_interfaces/TunableLaser/TunableLaser_Commands.xml

```diff
@@ -24,6 +24,18 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_command_changeAlignment</EFDB_Topic>
```

Comment:
Not sure this command name has the right connotation. How about `setOpticalConfiguration`?

Reply:
Shouldn't we rather use standard configuration sets? Make CSC configurable, and use the start command with configurationOverride to set the requested configuration set?

Reply:
I'm not 100% sure, but to only change optical configuration during the start command would necessitate cycling the CSC in the middle of a calibration script. I don't think this is ideal, especially since cycling the CSC will generate warnings (that the observers do need to be hearing).